src/Rinds/RindsSpecsDataSourcePoller.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Handles polling and storage of RINDS specs
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+namespace Automattic\WooCommerce\Admin\Rinds;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * RINDS specs data source poller class.
+ * This handles polling RINDS specs from JSON endpoints, and
+ * stores the specs in to the database as an option.
+ */
+class RindsSpecsDataSourcePoller {
+	const DATA_SOURCES = array(
+		'http://two.wordpress.test/rinds-specs.json',
+	);
+
+	/**
+	 * Polls the data sources for specs, persists those specs, and delete any
+	 * specs that no longer exist in the data sources.
+	 */
+	public static function poll_data_sources() {
+		$specs = array();
```

Comment:
What's this variable used for?  Will this be updated eventually?

Reply:
It's used to build up the specs and persist them as an option.